CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Change Log
 
+## [0.13.0](https://github.com/eclipse/lemminx/milestone/16?closed=1) (July 1, 2020)
+
+### Enhancements
+
+ * Grammar generator: generate a grammar from an XML document. See [#778](https://github.com/eclipse/lemminx/issues/778).
+ * Improved schema generation from an XML document whose name is also an extension. See [#805](https://github.com/eclipse/lemminx/issues/805).
+ * DTD generation: support for "optional" elements. See [#789](https://github.com/eclipse/lemminx/issues/789).
+ * Multiplicity for repeated elements in generated .XSD. See [#788](https://github.com/eclipse/lemminx/issues/788).
+ * Improved attribute generation for the grammar generator. See [#791](https://github.com/eclipse/lemminx/issues/791).
```

Comment:
I think we can remove the links on lines 8-11 because they address problems with the generator, which didn't exist in version 0.12.0. @fbricon could we remove these four?

Reply:
Yes I agree with @xorye

Reply:
Alright, I'll amend the PR and the vscode-xml PR to remove those lines.